cli/src/main/scala/com/lightbend/rp/reactivecli/argparse/kubernetes/KubernetesArgs.scala

```diff
@@ -79,6 +79,7 @@ case class KubernetesArgs(
   transformPodControllers: Option[String] = None,
   transformServices: Option[String] = None,
   namespace: Option[String] = None,
+  overrideVersion: Option[String] = None,
```

Comment:
I alluded to this above, but I'd rather this be part of `GenerateDeploymentArgs` as it is a general concern. Right now, we only have Kubernetes as a target but the plan is to add DC/OS as well, hence this distinction.